h2o-py/h2o/estimators/random_forest.py

```diff
@@ -481,18 +481,19 @@ def stopping_rounds(self, stopping_rounds):
     @property
     def stopping_metric(self):
         """
-        Metric to use for early stopping (AUTO: logloss for classification, deviance for regression). Note that custom
-        and custom_increasing can only be used in GBM and DRF with the Python client.
+        Metric to use for early stopping (AUTO: logloss for classification, deviance for regression and anonomaly_score
+        for Isolation Forest). Note that custom and custom_increasing can only be used in GBM and DRF with the Python
+        client.
 
         One of: ``"auto"``, ``"deviance"``, ``"logloss"``, ``"mse"``, ``"rmse"``, ``"mae"``, ``"rmsle"``, ``"auc"``,
-        ``"lift_top_group"``, ``"misclassification"``, ``"mean_per_class_error"``, ``"custom"``, ``"custom_increasing"``
-        (default: ``"auto"``).
+        ``"lift_top_group"``, ``"misclassification"``, ``"mean_per_class_error"``, ``"anomaly_score"``, ``"custom"``,
+        ``"custom_increasing"``  (default: ``"auto"``).
```

Comment:
Same point as Veronica has pointed out.  Please fix the gen_python.py gen_R.py to get rid of anomaly_score for all other algos except isolation forest.